--- origsrc/xnee-3.08/configure.in	2011-01-25 12:21:20.000000000 -0600
+++ src/xnee-3.08/configure.in	2011-02-09 01:58:02.564473000 -0600
@@ -29,7 +29,7 @@ LIBS="$LIBS $X_LIBS"
 
 
 AC_CHECK_LIB(c, dlopen, LIBDL="", [AC_CHECK_LIB(dl, dlopen, LIBDL="-ldl")])
-AC_CHECK_LIB(c, sem_init, LIBSEMA="CCC", [AC_CHECK_LIB(pthread, sem_init, LIBSEMA="-lpthread")])
+AC_CHECK_LIB(c, sem_init, LIBSEMA="", [AC_CHECK_LIB(pthread, sem_init, LIBSEMA="-lpthread")])
 AC_CHECK_LIB(Xtst, XTestFakeMotionEvent, LIBXTST="-lXtst",[AC_CHECK_LIB(Xext,XTestFakeMotionEvent , LIBXTST="-lXext")])
 
 GENERATION_DIRS=""
